### PR TITLE
Add instructions to connect to Capella

### PR DESCRIPTION
**Pull request by nithishr**
* Add connection string for Capella
* Add instructions about expectations

### CODE REVIEW

File: src/app.py [hunk 1]
```diff
@@ -48,8 +48,14 @@ def __init__(self, host, bucket, scope, collection, username, pw):
     def connect(self, **kwargs):
         # note: kwargs would be how one could pass in
         #       more info for client config
+
         conn_str = f"couchbase://{self.host}"
 
+        # Note: To connect with Couchbase Capella, use the following conn_str instead of the one above as it needs TLS
+        # Also ensure that the bucket, scopes & collections are created on Capella before running the application
+
+        # conn_str = f"couchbases://{self.host}?ssl=no_verify"
```
Comment (deniswsrosa) [inline src/app.py:57]: What if I want to use SSL?

**Review by deniswsrosa — CHANGES_REQUESTED** [review]

File: src/app.py (hunk 1 above)
Comment (nithishr) [reply src/app.py]: I have updated the code to work with SSL using certificates.

File: README.md [hunk 2]
```diff
@@ -30,6 +30,8 @@ All configuration for communication with the database is stored in the `.env` fi
 
 There is an example file `.env.example` that can be used as the template for the pararmeters for your environment. You can copy this file and fill in with the values corresponding to your environment.
 
+Note: If you are running with [Couchbase Capella](https://cloud.couchbase.com/), you need to change the connection string to use the TLS (currently commented out). Also ensure that the buckets, scopes & collections exist on the cluster and your [IP address is whitelisted](https://docs.couchbase.com/cloud/get-started/cluster-and-data.html#allowed) on the Cluster. If you want to use certificates, provide the path to the certificate in the environment variable, `CERT_PATH` and uncomment the ClusterOptions part of the code when connecting to the Cluster.
+
```
Comment (deniswsrosa) [inline README.md:34]: You are not specifying here how to get the certificate

**Review by deniswsrosa — CHANGES_REQUESTED** [review]
We need to also inform the user how to get the certificate, he probably has no idea which certificate you are talking about. Also make sure that you are adding a section in the tutorial talking about how to connect with capella

File: src/app.py [hunk 3]
```diff
@@ -91,6 +76,41 @@ def connect(self, **kwargs):
         except Exception as e:
             print(f"Error: {e}")
 
+    # Note: To connect with Couchbase Capella, use the following connect() instead of the one above as it communicates with TLS.
+    # This example does not use certificates for authentication. In production, you should have it enabled.
+    # Also ensure that the bucket, scopes & collections are created on Capella before running the application.
```
Comment (deniswsrosa) [inline src/app.py:81]: Nope, only the bucket should be present while using capela or a local server. In both cases we must automatically create the required indexes, scopes and collections.

**Review by deniswsrosa — CHANGES_REQUESTED** [review]
Only the bucket must be created before running the application. Scopes, Collections and indexes are automatically created by us.

**Pull request merged**